components/landing/highlights.json

```diff
@@ -60,7 +60,7 @@
       "title": "2016 wave of protests",
       "text": "Blocking of news websites and social media",
       "report": "https://ooni.io/post/ethiopia-report/",
-      "explore": "/search?input=ethsat.com&probe_cc=ET",
+      "explore": "/search?domain=ethsat.com&probe_cc=ET",
```

Comment:
This query is missing date ranges. cc @agrabeli

Reply:
Added.